packages/cli/src/components/IndexPane/IndexPane.tsx

```diff
@@ -0,0 +1,56 @@
+import Link from "next/link";
+import { ChangeEvent, useState } from "react";
+import ClientView from "./ClientView";
+import CompactView from "./CompactView";
+
+type IndexPaneProps = {
+  previews?: [string, string[]][];
+};
+
+const IndexPane: React.FC<IndexPaneProps> = ({ previews }) => {
+  const [compact, setCompact] = useState(true);
+
+  const handleToggle = (e: ChangeEvent<HTMLInputElement>) => {
+    console.log(e.target.value);
+    setCompact((current) => !current);
+  };
+
+  const showNullState =
```

Comment:
I find this state interesting, but don't have a good alternative suggestion. The logic seems brittle though

Reply:
yep, pulled from pages/index bc that will go away, but not sure about the good design / placement for it yet